### DISCUSSION TITLE
第1节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#49] [笔记]
12班的笔记交在这里哦

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/675926240

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec1/

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/WO1LdXj81obu4jxUmUzcxny7nvc?from=from_copylink

**Comment by ltstar** [comment]
[课程1.md](https://github.com/InternLM/tutorial/files/13825088/1.md)

**Comment by siuserjy** [comment]
https://blog.csdn.net/weixin_46334272/article/details/135374581

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135375187

**Comment by wincharm001** [comment]
https://zhuanlan.zhihu.com/p/676014254

**Comment by Jeff-Huang-SHU** [comment]
https://www.yuque.com/u34036176/dxc2up/gpco7ac28pprx1wm?singleDoc# 《note1》

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/IHM9d1TWuo7e7zxSFugcAYCNn8f

**Comment by XiLinky** [comment]
https://blog.csdn.net/m0_73202283/article/details/135405586

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17944363

**Comment by hblovo** [comment]
[Lesson1.md](https://github.com/InternLM/tutorial/files/13849761/Lesson1.md)

**Comment by hxy-tsy** [comment]
[书生浦语大模型.md](https://github.com/InternLM/tutorial/files/13849911/default.md)

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135447026

**Comment by Silflame** [comment]
https://www.yuque.com/wayne-o8axv/hdcch1/lb3vgg9yqdlgp1ng

**Comment by ShenFeilang** [comment]
https://ink-neon-bce.notion.site/InternLM-3a05ca3720dd4ef3a024c68c3a9b808c?pvs=4
